conf/app.ini

```diff
@@ -116,6 +116,8 @@ SSH_LISTEN_HOST = 0.0.0.0
 SSH_LISTEN_PORT = %(SSH_PORT)s
 ; Root path of SSH directory, default is '~/.ssh', but you have to use '/home/git/.ssh'.
 SSH_ROOT_PATH =
+; Choose the ciphers to support for SSH connections
+SSH_SERVER_CIPHERS = aes128-ctr, aes192-ctr, aes256-ctr, aes128-gcm@openssh.com, arcfour256, arcfour128
```

Comment:
Hi Unknwon - nice to meet you!

Issue #4107 contains an excerpt from the go source for the ssh package, this is the reference i have been working with, i presumed it to be comprehensive.

Let me know if theres anything else and please see revised commit below